mixer/test/client/env/setup.go

```diff
@@ -155,18 +166,16 @@ func (s *TestSetup) ReStartEnvoy() {
 
 // VerifyCheckCount verifies the number of Check calls.
 func (s *TestSetup) VerifyCheckCount(tag string, expected int) {
-	if s.mixer.check.Count() != expected {
```

Comment:
why these needs to have an eventual waiting loop, can we notify the verification by blocking on channels?

Reply:
Not without large changes across the mixer test suite.